search_func/Source/utils.cpp

```diff
@@ -0,0 +1,29 @@
+#include "utils.hpp"
+
+
+int find(std::vector<std::string> vector, std::string value)
+{
+    int i;
+	for(i = 0; i < (int)vector.size(); ++i)
+	{
+		if(vector[i] == value)
+		{
+			break;
+		}
+	}
+	if(i > (int)vector.size()) return -1;
+	return i;
+}
+
+
+std::string toString(std::size_t x)
```

Comment:
nazywaj zmienne. x mało mówi. 

A co ta funcja ma robić ? Bo istnieje funcja std::to_string która zamienia wartość na stringa

Reply:
No właśnie miała robić to samo, tylko że nie wiedziałam, że jest coś takiego jak std::to_string, a nie szukałam w necie, bo w sumie to tylko parę linijek. Ale dzięki, zmienię w następnym commicie.